addons/backpacks/functions/fnc_backpackOpened.sqf

```diff
@@ -29,8 +28,8 @@ _sounds = [
     "a3\sounds_f\characters\ingame\AinvPpneMstpSlayWpstDnon_medic.wss",
     "a3\sounds_f\characters\ingame\AinvPpneMstpSlayWrflDnon_medic.wss"*/
 
-    QUOTE(PATHTO_R(sounds\zip_in.wav)),
-    QUOTE(PATHTO_R(sounds\zip_out.wav))
+    QUOTE(PATHTOF(sounds\zip_in.wav)),
```

Comment:
These are not equivalent
"z\ace\addons\backpacks\sounds\zip_out.wav" vs "\z\ace\addons\backpacks\sounds\zip_out.wav"

`say3d` is very picky and won't accept PATHTOF